feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go

```diff
@@ -373,10 +381,19 @@ func TestTrafficWithGracefulRestartSpeaker(t *testing.T) {
 		t.Log("Start DUT interface Config")
 		configureDUT(t, dut)
 	})
+
+	t.Log("Configure Network Instance")
+	d := &telemetry.Device{}
+	ni := d.GetOrCreateNetworkInstance(*deviations.DefaultNetworkInstance)
```

Comment:
Do we need to create the default network instance or should it already exist?

If it already exists we can only change the **RouterId** of the default network instance.
```
dutNI := dc.NetworkInstance(*deviations.DefaultNetworkInstance)
dutNI.RouterId().Replace(t, dutDst.IPv4)
```

This snippet can be added as part of **configureDUT()** instead.